src/devtools/client/debugger/src/components/Editor/ToggleWidgetButton.tsx

```diff
@@ -182,7 +177,7 @@ function ToggleWidgetButton({ editor, cx, breakpoints }: ToggleWidgetButtonProps
       editor.codeMirror.off("lineMouseEnter", onLineEnter);
       editor.codeMirror.off("lineMouseLeave", onLineLeave);
     };
-  }, []);
+  }, [editor]);
```

Comment:
I don't think this is the right change here. We're referencing functions `onLineEnter` and `onLineLeave` that are declared in the render body, but we're only re-running the effect when `editor` changes. This likely means the functions are going to be stale.

It shouldn't matter in this specific case since they close over state update functions (which are themselves stable) but it seems like a future foot-gun.